catalog/ui/search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars

```diff
@@ -208,7 +208,7 @@
 
                         <div class="input-group input-group-sm">
                             <span class="input-group-addon">Radius&nbsp;</span>
```

Comment:
The `name="radiusValue"` was removed in DDF-1539 to fix a bug with the radius value being improperly reset. Can't you just select this `input` by id anyway?